en/admin/migration.inc

```diff
@@ -115,6 +115,10 @@ The following is [options] used in **cubrid unloaddb**.
     
     This option specifies the name of the hash file.
 
+.. option:: --latest-image
+   
+    This option makes it dump the latest image of objects in data volumes when dumping objects, not referring to log volumes (ignoring MVCC). It could include uncommitted data and deleted data which has not vacuummed.
```

Comment:
```suggestion
    This option makes it dump the latest image of objects in data volumes when dumping objects, not referring to log volumes (ignoring MVCC). It could include uncommitted data and deleted data which has not been vacuummed.
```
I think 'been' is need to be added.

Reply:
```suggestion
    This option unloads the latest image of objects in data volumes when unloading instances (the MVCC version is ignored and does not refer to the log volumes). It may include uncommitted data and deleted data that has not been vacuummed.
```

Reply:
@charischau This utility unloads even without this option, but this option control how it unloads. In this respect, "This option **unloads** the latest image" looks a little awkward. What do you think?

Reply:
okay, I understand. 
Then I would suggest changing the sentence back to ' This option make it to unload the latest image...'. 

```suggestion
    This option makes it to unload the latest image of objects in data volumes when unloading instances (the MVCC version is ignored and it does not refer to the log volumes). It may include uncommitted data and deleted data that has not been vacuumed.
```

Reply:
Thank you for review.